### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by zekabra** [sunilpaulmathew/Translator#34]
Good app, but there is an option missing that would help a lot for those who are translating, showing the string above the text to be translated, as it only shows the text to be translated, in cases where there is doubt in the translation because the string is not visible, we lose the visibility of information that could help us decide the correct translation

**Comment by franco27** [comment]
Yes, it's true that it would be better to have the source text at the top and the translated text at the bottom.

**Comment by sunilpaulmathew** [comment]
@franco27 
I'll look on it. Thanks for the suggestion.
